eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```diff
@@ -302,6 +302,9 @@ services:
             - $content.default_ttl$
         tags:
             - { name: kernel.event_subscriber }
+        calls:
+            - [setConfigurator, ['@ezplatform.http_cache.response_configurator']]
+            - [setDispatcherTagger, ['@ezplatform.http_cache.response_tagger.dispatcher']]
```

Comment:
Because I didn't want to add them as constructor arguments... 'cause of bit of BC, and because of quantum.

But I agree that this isn't ideal. I had two options in mind:
a) Deprecate this listener (and untag the service ? what is BC supposed to be like on a listener ?), and write a new one
b) add the services at the end in the constructor, and ignore that it is a ugly
c) ... ?

I'm open to suggestions, because I would like to get rid of these calls. In any case, it's not enough even for BC, as I don't test if the configurator is there or not. If it is not, we need to tag using the old approach for BC.

Or am I overthinking this ?